alchemy/src/cloudflare/scope-extensions.ts

```diff
@@ -0,0 +1,16 @@
+import type { CloudflareApiOptions } from "./api.ts";
+
+/**
+ * Cloudflare scope extensions - adds Cloudflare credential support to scope options.
+ * This uses TypeScript module augmentation to extend the ProviderCredentials interface.
+ */
+declare module "../scope.ts" {
+  interface ProviderCredentials {
+    /**
+     * Cloudflare credentials configuration for this scope.
+     * All Cloudflare resources created within this scope will inherit these credentials
+     * unless overridden at the resource level.
+     */
+    cloudflare?: CloudflareApiOptions;
+  }
+}
```

Comment:
Are these respected? I don't see any updates to `createCloudflareApi`

Reply:
Nope, not being used anywhere. I don't have a cloudflare account. So I just added them. Someone else will probably have to use them and test them. I didn't want to break anything.